tests/test_bible.py

```diff
@@ -28,6 +31,11 @@ def test_includes(self):
             # acts 2 has 47 verses
             self.two_books.includes(bible.Verse('Acts 2:48'))
 
+        """ TODO: test for range of boxed/omitted passages        
+        with self.assertRaises(RangeError):
+            # mark 16:9-20 has been boxed/omitted
+            self.boxed.includes(bible.Verse('Mark 16:9')) """
+
```

Comment:
Why is this commented out?

Reply:
Removed lines for now. It's a TODO for now. Will work on RangeError exception class soon